lib/cloud_controller.rb

```diff
@@ -8,6 +8,8 @@
 
 require 'uaa/token_coder'
 
+module VCAP; end
```

Comment:
What's this for?

Reply:
Previously, RingBuffer was the first file loaded with the `VCAP` module defined:
- https://github.com/cloudfoundry/cloud_controller_ng/blob/df570cbf20aea26406ec9678e5e848a156a81dc6/lib/cloud_controller.rb#L11
- https://github.com/cloudfoundry/cloud_controller_ng/blob/df570cbf20aea26406ec9678e5e848a156a81dc6/lib/sinatra/vcap.rb#L1
Removing RingBuffer has the side effect of causing `http_constants.rb` to fail loading because `VCAP` is undefined.
- https://github.com/cloudfoundry/cloud_controller_ng/blob/df570cbf20aea26406ec9678e5e848a156a81dc6/lib/sinatra/vcap.rb#L2
- https://github.com/cloudfoundry/cloud_controller_ng/blob/df570cbf20aea26406ec9678e5e848a156a81dc6/lib/vcap/rest_api.rb#L1
- https://github.com/cloudfoundry/cloud_controller_ng/blob/df570cbf20aea26406ec9678e5e848a156a81dc6/lib/vcap/rest_api/http_constants.rb#L1

Reply:
So, rather than shifting module definitions around for what random file gets to be the first one to define VCAP, I pulled it up to the top level.